### ISSUE TITLE
Feature Request: Search in english and french at the same time

### ISSUE THREAD

**Issue by Nosirus** [nosmokingbandit/Watcher3#261]
To be able to add with the French metadata of TMDB and that it looks with the title English and French at the same time.

The "Preferred Words" will make the difference during the search with the tags "french, truefrench, etc ..." many team do not necessarily put the names in French for a French release.

I think a lot of French (or other language) user will like this option ! me yes anyway it's all that I miss him :D

**Comment by nosmokingbandit** [comment]
Give me an example of a movie title like this.

**Comment by Nosirus** [comment]
Of course, the movie "Star Wars: The Last Jedi" can be found with this title or the French title "Star Wars 8 : Les Derniers Jedi" or the opposite, the movie "Les Gardiens de la Galaxie Vol. 2" in French title and better quality and more easily found with the original title "Guardians of the Galaxy 2"

often it is the same release made by the same team in multi language but those who post the torrent, decides to use the original name or translated

**Comment by nosmokingbandit** [comment]
This can get tricky so I may need to pick you brain a bit. Living in an English-speaking country means I don't often have to think about these things.

The actual problem in question is that torrent/nzb releases aren't being accepted by Watcher because the titles don't match, right? The logs should indicate this when you force a search for that movie. If that is the case, do you get the desired releases by disabling `Score and filter release titles` in Settings/Quality under the quality profile for the movies? Disabling this is not a good long-term fix since you could end up with all sorts of downloads you don't want, but if this corrects the problem I can have the database store non-English titles to match against as well so later we can leave that option enabled while still being able to download movies in French, Spanish, or whatever.
